src/hooks/useNetwork.ts

```diff
@@ -29,5 +29,5 @@ export default function useNetwork({onReconnect = () => {}}: UseNetworkProps = {
         prevOfflineStatusRef.current = isOffline;
     }, [isOffline]);
 
-    return {isOffline: isOffline ?? false};
+    return {isOffline: networkStatus === CONST.NETWORK.NETWORK_STATUS.OFFLINE};
```

Comment:
What does this do? It's kind of weird that we have an `isOffline` variable, but we don't use it here and are instead looking at the status. 

It looks in the other components, you're using the `networkStatus` variable to make decisions, so why does this need to change?

Reply:
Yes, I understand that. But the logic here seems unintuitive to me. That we have an `isOffline` variable. But then we return `isOffline` as something other than that variable. 

Is there a way that we can make this more clear and easy to understand?